### PR TITLE
Disabled warnings for using Climates of Tamriel with Sounds of Skyrim…

### PR DESCRIPTION
**Pull request by Velgus**
… if using Weather and Ambience Overhaul

### CODE REVIEW

**Comment by Freso** [comment]
I'm a bit out of the loop with Skyrim modding. Do you have a link to a source for this?

**Comment by Velgus** [comment]
[Weather and Ambience Overhaul](http://www.nexusmods.com/skyrim/mods/55705/?) basically is an all in one patch for compatibility between a variety of weather and audio mods. On that page, under the "Redundancy List", both the patches I removed the warnings for are considered redundant.

**Comment by Freso** [comment]
Indeed. There should probably be added another message (possibly just `inc` entries) that the plugins aren't needed with "WAO-WeatherAndAmbienceOverhaul.esp" enabled though.

**Comment by WrinklyNinja** [comment]
> Indeed. There should probably be added another message (possibly just inc entries) that the plugins aren't needed with "WAO-WeatherAndAmbienceOverhaul.esp" enabled though.

I agree. If @Velgus adds `inc` entries for `WAO-WeatherAndAmbienceOverhaul.esp`, then this can be merged.

**Comment by Velgus** [comment]
Added them.

File: masterlist.yaml [hunk 1]
```diff
@@ -28356,3 +28358,9 @@ plugins:
     after:
       - 'AOS.esp'
     url: [ 'http://www.nexusmods.com/skyrim/mods/67559' ]
+  - name: 'WAO-WeatherAndAmbienceOverhaul.esp'
+    inc:
```
Comment (Freso) [inline masterlist.yaml]: Please take care to not leave trailing whitespaces. ;)

File: masterlist.yaml [hunk 2]
```diff
@@ -28356,3 +28358,9 @@ plugins:
     after:
       - 'AOS.esp'
     url: [ 'http://www.nexusmods.com/skyrim/mods/67559' ]
+  - name: 'WAO-WeatherAndAmbienceOverhaul.esp'
```
Comment (Freso) [inline masterlist.yaml]: It would be nice if you would add an `url` here too. :)

**Comment by Freso** [comment]
Two minor nitpicks and I'll be happy. 👍 😉

**Comment by Velgus** [comment]
Apologies in both cases, fixed.

**Comment by Freso** [comment]
No need to apologise. Just consider it a learning experience. :)

**Pull request merged**